transactions/update-contract/2025/feb-10-migration-contract-stub/README.md

```diff
@@ -0,0 +1,31 @@
+# Update FlowServiceAccount Contract
```

Comment:
Shouldn't this be "Deploy Migration stub smart contract"?

Reply:
you are right :(

Reply:
changed it to deploy contract.